### ISSUE TITLE
It seems that the lr0 parameter specified during training is ignored and invalid. What should I do?

### ISSUE THREAD

**Issue by tms2003** [ultralytics/ultralytics#8400]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I want to fine-tune my training results, my training code is as follows:

```

yolo detect train data=data/coco11.yaml model=/yolov8/runs/detect/train5/weights/best.pt epochs=5 imgsz=640  batch=64 patience=0  device=0,1,2,3   lr0=0.0001 freeze=10
```

The prompt reads:
```

optimizer: 'optimizer=auto' found, ignoring 'lr0=0.0001' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically
```
It seems that the lr0 parameter specified during training is ignored and invalid. What should I do?

**Comment by Burhan-Q** [comment]
@tms2003 you need to specify an optimizer, when `optimizer="auto"` (default) the learning rate `lr0` and `momentum` arguments are ignored.

https://github.com/ultralytics/ultralytics/blob/21088e3986971a596fc3149ec017921eb2ce93c8/ultralytics/engine/trainer.py#L711-L716
